src/Data/Aeson/KeyMap.hs

```diff
@@ -85,6 +85,11 @@ module Data.Aeson.KeyMap (
     filterWithKey,
     mapMaybe,
     mapMaybeWithKey,
+    
+    -- * Key Type
+    Key,
+    fromText,
```

Comment:
I'm fine with re-exporting `Key`, but `toText` and `fromText` read wrong:

```haskelll
key = KeyMap.fromText t
```

Reply:
Done!